cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl":"http://172.19.0.2:5000"
```

Comment:
I don't like idea of putting values like that in your code. Maybe `.env.` would be a better place? Plus this JSON is not formatted properly. ;)